sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp

```diff
@@ -12,9 +12,13 @@
 #include "azure/storage/blobs/blob_client.hpp"
 #include "azure/storage/blobs/blob_container_client.hpp"
 #include "azure/storage/blobs/blob_lease_client.hpp"
+#include "azure/storage/blobs/blob_options.hpp"
+#include "azure/storage/blobs/blob_responses.hpp"
 #include "azure/storage/blobs/blob_sas_builder.hpp"
 #include "azure/storage/blobs/blob_service_client.hpp"
 #include "azure/storage/blobs/block_blob_client.hpp"
 #include "azure/storage/blobs/dll_import_export.hpp"
 #include "azure/storage/blobs/page_blob_client.hpp"
-#include "azure/storage/common/storage_exception.hpp"
+#include "azure/storage/blobs/rest_client.hpp"
+#include "azure/storage/blobs/rtti.hpp"
```

Comment:
I don't think we should expose `rest_client.hpp`. It may be renamed in the future. I think we should only expose it indirectly, like via `blob_options.hpp`.

Reply:
If it gets renamed, the all-inclusive header should get updated too. As I said above, the goal is to include all public headers. If customer can see it intellisense, when typing `#include <azure/storage/blobs/`, it should be included in this file.